src/proxy-ui-api/src/test/java/org/niis/xroad/restapi/config/audit/MockableAuditEventLoggingFacade.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * The MIT License
+ * Copyright (c) 2018 Estonian Information System Authority (RIA),
+ * Nordic Institute for Interoperability Solutions (NIIS), Population Register Centre (VRK)
+ * Copyright (c) 2015-2017 Estonian Information System Authority (RIA), Population Register Centre (VRK)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.niis.xroad.restapi.config.audit;
+
+import org.niis.xroad.restapi.util.RequestHelper;
+import org.niis.xroad.restapi.util.SecurityHelper;
+import org.niis.xroad.restapi.util.UsernameHelper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+
+@Component
+@Profile({ "test"})
+public class MockableAuditEventLoggingFacade extends AuditEventLoggingFacade {
```

Comment:
MockableAuditEventLoggingFacade exists to make `callAuditLoggerLogSuccess` and `callAuditLoggerLogFailure` public, so that tests can `verify()` them. It's maybe not the perfect design, and there could be alternatives, like moving `AuditLoggingRestTemplateTest` to package `org.niis.xroad.restapi.config.audit`....but maybe I will keep it like it is?

Reply:
Yes there are the usual multiple alternatives how to achieve this. I don't see so much benefit in changing the current implementation so we can keep it this way 👍